Setup/env_test.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/sudo /bin/bash
+
+LOGFILE="${LOGFILE:-./env_test.log}"
+URLS_TO_CHECK='http://www.google.com/ https://www.google.com/ http://www.ericom.com/ https://www.ericom.com/ https://hub.docker.com/'
+SHIELD_NETWORK_ADDR_BLOCK='10.20.0.0/16'
+
+if ! declare -f log_message >/dev/null; then
+    function log_message() {
+        echo "$1"
+        echo "$(date): $1" >>"$LOGFILE"
+    }
+fi
+
+if ! declare -f failed_to_install >/dev/null; then
+    function failed_to_install() {
+        log_message "An error occured during the installation: $1, Exiting!"
```

Comment:
we want to use this script as a standalone (pre-install)
so the correct message should be:
"Ericom Shield cannot be install on this environment: $1, Exiting"

Reply:
Corrected. Thanks!